src/StackManagerBundle/Mapper/StackApiMapper.php

```diff
@@ -44,11 +45,42 @@ public function __construct(
      */
     public function create($name)
     {
-        $stack = current($this->cloudFormationClient->DescribeStacks([
+        $stack = $this->createFromApiResponse(current($this->cloudFormation->DescribeStacks([
             'StackName' => $name,
-        ])->get('Stacks'));
+        ])->get('Stacks')));
 
-        foreach ($stack['Tags'] as $tag) {
+        return $stack;
+    }
+
+    /**
+     * Get Stack models for all stacks with an appropriate template and
+     * environment tag using the AWS CloudFormation API.
+     *
+     * @return array Stack models
+     */
+    public function findAll()
+    {
+        $response = $this->cloudFormation->DescribeStacks();
+        foreach ($response['Stacks'] as $data) {
+            $stack = $this->createFromApiResponse($data);
+            if ($stack) {
+                $stacks[] = $stack;
+            }
+        }
+
+        return $stacks;
```

Comment:
Will this cause an issue if there are no stacks in the ```DescribeStacks()``` response - will ```$stacks``` not have been initialised?